workflows/VGP-assembly-v2/Assembly-Hifi-HiC-phasing-VGP4/Assembly-Hifi-HiC-phasing-VGP4-tests.yml

```diff
@@ -1,14 +1,14 @@
 - doc: Test outline for Assembly-Hifi-HiC-phasing-VGP4
   job:
-    Hi-C Forward reads:
+    Hi-C reads - Forward:
```

Comment:
That seemed better before ? Naming things is definitely the hardest part about maintaining IWC workflows 😅